example-android-kts/app/build.gradle.kts

```diff
@@ -30,24 +29,13 @@ android {
     sourceSets {
         getByName("main").java.srcDirs("src/main/kotlin")
     }
-
-    lintOptions {
-        setAbortOnError(false)
-    }
 }
 
 dependencies {
-    implementation("androidx.appcompat:appcompat:1.2.0")
-    testImplementation("junit:junit:4.13")
+    implementation("androidx.appcompat:appcompat:1.3.1")
+    testImplementation("junit:junit:4.13.2")
 }
 
-license {
-    header = file("../../LICENSE_HEADER.txt")
-    strictCheck = true
-}
```

Comment:
Applied globally somewhere, or you don't care about attributing this example any more?

Reply:
This plugin doesn't support Gradle 7 and I kind of just stopped caring enough to hold off until they support Gradle 7

Reply:
Ahh, gotcha. 👍🏻